en/graph-views/secure-your-map-platform.md

```diff
@@ -130,6 +130,11 @@ procedure](advanced-configuration.html#change-centreon-map-server-port).
 
 > Don't forget to modify the URL on Centreon side in **Map server address**
 > field in `Administration > Extensions > Map > Options` menu.
+![image](../assets/graph-views/map-address-https.png)
+
+> Don't forget to update your connection profile in the desktop client 
+> by checking the ssl checkbox.
+![image](../assets/graph-views/desktop-ssl-option.png)
```

Comment:
screenshot is in French

Reply:
Unfortunately, you can't change the language in the desktop.